scripts/post-install-linux.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh -e
+
+nixbld_user=nixbld
+nixbld_group=nixbld
+storedir="$(nix-store --verbose --version 2>/dev/null | sed -n 's/^Store directory: //p')"
+localstatedir="$(nix-store --verbose --version 2>/dev/null | sed -n 's/^State directory: //p')"
+
+
+# Setup build users
+if ! getent group "$nixbld_group" >/dev/null; then
+  groupadd -r "$nixbld_group"
+fi
+
+for i in $(seq 10); do
```

Comment:
The default of 10 is insufficient for modern systems. NixOS creates 32 build users.

Reply:
Why not just compute the number? A mathematical upper bound is the multiplication of all constants in nix.conf. What the lowest upper bound is, is an exercise left for the reader. 

Hard coding a constant to work for all systems won't work in general.

Reply:
Missed this one. I've bumped the limit to 32.

@williammpratt We can't really compute it anyhow -- this number specifies (indirectly) maximum number of concurrent builds, because each build is run from its own user. Therefore the only requirement for this number is to be bigger or equal than `max-jobs` and even that may be not enough. 32 is big enough for most cases so we use it.